core/systems/input/focus_manager.gd

```diff
@@ -46,6 +46,8 @@ func _on_child_tree_changed(_node) -> void:
 	for child in parent.get_children():
 		if not child is Control:
 			continue
+		if not child.is_inside_tree():
```

Comment:
This seems unrelated? If not related to logging can you document it in the commit message?